index.html

```diff
@@ -73,8 +73,638 @@
+                    <p>
+                      Featuring a unique history, uninhibited wildness and at times
+                      otherworldly natural spectacles, Hokkaido offers an experience
+                      that is completely different from the rest of Japan.
+                    </p>
+                  </div>
+                </div>
+                <div>
+                  <div class="img-box">
+                      <a href="#"><img class="feat-img" src="./images/insights4.png" alt=""/></a>
+                    <div class="bottom-left nowrap">
+                      <p class="bottom-left text-primary fw-bold bg-grey p-2 ">SPONSORED INSIGHTS</p>
+                    </div>
+                  </div>
+                  <div class="img-text">
+                    <h5><a class="fw-bold text-decoration-none text-dark" href="#">Hokkaido: Japan’s Northern Frontier</a></h5>
+                    <p>
+                      Featuring a unique history, uninhibited wildness and at times
+                      otherworldly natural spectacles, Hokkaido offers an experience
+                      that is completely different from the rest of Japan.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+        </div>
+        <div class="p-2 col-lg-4 col-md-6 col-sm-12 content-2">
+             <div class="p-2 mb-2 d-sm-none d-md-block" id="top-story">
+              <div>
+                <h6 class="text-secondary">TOP STORY</h6>
+                <div class="img-box">
+                    <a href="#"><img class="feat-img" src="./images/biden2.png" alt=""/></a>
+                  <div class="bottom-left">
+                    <p><a class="text-danger fw-bold text-decoration-none bg-grey p-2" href="#">POLITICS</a></p>
+                  </div>
+                </div>
+                <div class="img-text">
+                  <h5>
+                    <a class="fw-bold text-decoration-none text-dark" href="#"
+                      >Joe Biden's First Executive Actions Undo Many of Donald
+                      Trump's</a
+                    >
+                  </h5>
+                  <p>
+                    Actions related to the COVID-19 crisis, climate change and
+                    racial equality are to be among the first taken as he begins his
+                    tenure.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="p-2 mb-2" id="my-turn">
+              <section>
+                <h6 class="text-secondary">MY TURN</h6>
+                <div>
+                  <div class="myturn d-md-flex justify-n">
+                    <div>
+                      <a href="#" class="text-decoration-none">
+                        <img class="l-img" src="./images/myturn1.jpg" alt="" />
+                        <p class="fw-bold text-secondary">'I've Rescued Hundreds of Special Needs Dogs'</p>

```

Comment:
- [ ]  Every title in this category should be an `h` tag. Please use an` h` tag as well for other titles in this section.